internal/server/statvar/golden/search_statvar_test.go

```diff
@@ -36,102 +36,88 @@ func TestSearchStatVar(t *testing.T) {
 
 	testSuite := func(mixer pb.MixerClient, recon pb.ReconClient, latencyTest bool) {
 		for _, c := range []struct {
-			query           string
-			places          []string
-			enableBlocklist bool
-			svOnly          bool
-			goldenFile      string
+			query      string
+			places     []string
+			svOnly     bool
+			goldenFile string
 		}{
 			{
 				"Asian , age",
 				[]string{"geoId/06"},
 				false,
-				false,
 				"asian_age.json",
 			},
 			{
 				"crime",
 				[]string{},
 				false,
-				false,
 				"crime.json",
 			},
 			{
 				"female",
 				[]string{"country/USA"},
 				false,
-				false,
 				"female.json",
 			},
 			{
```

Comment:
can remove this test and the test file

Reply:
wait, why would I remove this test file? this file tests for the search results retrieved given a query ..